src/components/CustomerAppeal/ReasonSelection/ReasonSelectionHeader.test.tsx

```diff
@@ -0,0 +1,20 @@
+import { render, cleanup } from "@testing-library/react-native";
+import React from "react";
+import { ReasonSelectionHeader } from "./ReasonSelectionHeader";
+import "../../../common/i18n/i18nMock";
+
+describe("ReasonSelectionHeader", () => {
+  afterEach(() => {
+    cleanup();
+    jest.resetAllMocks();
+  });
+
+  it("should render the header correctly", async () => {
+    expect.assertions(1);
+    const { queryByText } = render(
+      <ReasonSelectionHeader title="Reason Header" />
```

Comment:
just curious, what happens if the title with empty string

Reply:
The header DOM will have `""` as the value

Reply:
thanks.. so the `expect(...)` is going to null.. can include that also